.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,7 +3,7 @@
 
 #### Where should the reviewer start?
 
-#### How can this be manually tested? (brief repro steps)
+#### How can this be manually tested? (brief repro steps and URL with change)
```

Comment:
Although agree we should share the environment urls, I am not bought into the idea of sharing the dev urls publicly esp. for dev env. at this point. 
It seems there is actually a way to lock test/dev environments, but we have not done that yet. And not everybody is using internal VMs at this point.

Reply:
That comment goes with all your PRs at this point - don't want to copy paste to all :)

Reply:
great point @kevsers and thanks for taking the time to chat with me about this in person. i'll update the phrasing for this and the other similar changes.